osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/WatchHighScoresCommand.cs

```diff
@@ -133,7 +133,7 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
                                                  }))
                                              // there might be multiple queue entries for the same insert. this can cause issues due to how we do the mapping lookup so let's fix that.
                                              .DistinctBy(s => s.score_id)
-                                             .OrderBy(s => s.queue_id)
+                                             .OrderBy(s => s.score)
```

Comment:
uhhhhhhhhhhhhhhhhhhhhhhhhhhh

```suggestion
                                             .OrderBy(s => s.score_id)
```